examples/atlas-api-key/create-api-key-assign-to-multiple-projects/versions.tf

```diff
@@ -2,7 +2,7 @@ terraform {
   required_providers {
     mongodbatlas = {
       source  = "mongodb/mongodbatlas"
-      version = "~> 1.0"
+      version = "~> 1.10.0"
```

Comment:
Is our intention here to leave a fixed minor version? With the previous definition, any new minor version would be adopted: https://developer.hashicorp.com/terraform/language/expressions/version-constraints#-3.

Reply:
If we don't provide a version, `tflint` will complain. In this case, when running tf validate locally with `~>1.0`, terraform downloaded a provider version without the feature used in the example. Here I am adding a version that contains the feature used in the example so customers will not hit the same issue.